sender_report.go

```diff
@@ -96,19 +101,22 @@ func (r SenderReport) Marshal() ([]byte, error) {
 	binary.BigEndian.PutUint32(packetBody[srPacketCountOffset:], r.PacketCount)
 	binary.BigEndian.PutUint32(packetBody[srOctetCountOffset:], r.OctetCount)
 
-	for i, rp := range r.Reports {
+	offset := srHeaderLength
+	for _, rp := range r.Reports {
 		data, err := rp.Marshal()
 		if err != nil {
 			return nil, err
 		}
-		offset := srHeaderLength + receptionReportLength*i
 		copy(packetBody[offset:], data)
```

Comment:
Is this offset math safe (should we assert anything?)

Reply:
I'll take a look in the morning and merge then. I wrote this late last night so there could be a bug.